charts/federation-v2/charts/controllermanager/templates/role.yaml

```diff
@@ -12,31 +12,69 @@ rules:
 - apiGroups:
   - scheduling.federation.k8s.io
   resources:
-  - '*'
```

Comment:
For federation api groups, I think `*` is preferable.  Otherwise maintenance will be required, and I don't think there is any advantage to requiring that.

Reply:
Actually, when I'm doing this, I'm thinking it is a balance between convenience and security. My preferred solution is to use * in `resources` and `verbs` for all api groups created by federation because it seems less peopole will reuse federation groups for their own purpose. So, it is convenient in extendibility for federation controller manages their own resource groups with full permission. 

However, I'm not full of knowledge about K8s RBAC. And strict checking is fine for most of case. So, I follow the rules metioned in #687 .

For this comments, did you mean for all api groups? If true, what is the purpose to metion each resource independently in #687, I'm a little confused.

Reply:
I meant that `*` should be used for resources for all federation groups.

Reply:
all federated resource groups with resource '*'. `clusterregistry.k8s.io` is an exception because we only need `cluster` in this independent maintained group.

Reply:
@marun